locale/ca.po

```diff
@@ -266,6 +278,22 @@ msgctxt "model:ir.message,text:msg_no_session"
 msgid "Session not found."
 msgstr "Sessió no trobada."
 
+msgctxt "model:ir.message,text:msg_origin_line_with_move"
+msgid ""
+"The line have a move %(move)s related. Delete the line will create a cancell"
+" move. Do you want to continue?"
+msgstr ""
+"La línia té un assentament %(move)s relacionat. Suprimir la línia crearà un "
+"assentament de cancel·lació. Vols continuar?"
+
+msgctxt "model:ir.message,text:msg_origin_pending_amount"
+msgid ""
+"The origin could not be conciled becasue the lines amount is "
```

Comment:
conciled -> reconciled
becasue -> because

The origin could not be reconciled because the amount of the lines "%(line_amount)s" and the amount of the origin "%(origin_amount)s" differ.